src/main/scala/com/goyeau/kubernetes/client/KubernetesClient.scala

```diff
@@ -26,10 +26,10 @@ case class KubernetesClient[F[_]: ConcurrentEffect](httpClient: Client[F], confi
 }
 
 object KubernetesClient {
-  def apply[F[_]: ConcurrentEffect](config: KubeConfig): Resource[F, KubernetesClient[F]] =
+  def apply[F[_]: ConcurrentEffect](config: KubeConfig)(implicit cs: ContextShift[IO]): Resource[F, KubernetesClient[F]] =
     BlazeClientBuilder[F](ExecutionContext.global, Option(SslContexts.fromConfig(config))).resource
       .map(httpClient => apply(httpClient, config))
 
-  def apply[F[_]: ConcurrentEffect](config: F[KubeConfig]): Resource[F, KubernetesClient[F]] =
+  def apply[F[_]: ConcurrentEffect](config: F[KubeConfig])(implicit cs: ContextShift[IO]): Resource[F, KubernetesClient[F]] =
```

Comment:
```suggestion
  def apply[F[_]: ConcurrentEffect: ContextShift](config: F[KubeConfig]): Resource[F, KubernetesClient[F]] =
```
With this we are not forcing the IO monad

Reply:
No need anymore. Changed to `Async`